### PR TITLE
Add Organization crud template

### PR DESCRIPTION
**Pull request by Krishanx92**
## Purpose

CREATE TABLE IF NOT EXISTS ORGANIZATION (
        UUID VARCHAR(50),
        NAME VARCHAR(255),
		DISPLAY_NAME VARCHAR(255),
        PRIMARY KEY(UUID)
        );
		
CREATE TABLE IF NOT EXISTS ORGANIZATION_CLIAM_MAPPING (
        UUID VARCHAR(50),
        CLAIM_KEY VARCHAR(255),
		CLAIM_VALUE VARCHAR(255),
		FOREIGN KEY(UUID) REFERENCES ORGANIZATION(UUID) ON UPDATE CASCADE ON DELETE CASCADE,
        UNIQUE(UUID,CLAIM_KEY)
        );
		
CREATE TABLE IF NOT EXISTS ORGANIZATION_NAMESPACE_MAPPING (
        UUID VARCHAR(50),
        NAMESPACE VARCHAR(255),
		FOREIGN KEY(UUID) REFERENCES ORGANIZATION(UUID) ON UPDATE CASCADE ON DELETE CASCADE,
        UNIQUE(UUID,NAMESPACE)
        );

Screenshot from 2023-02-09 10-25-58

### CODE REVIEW

File: admin/admin-domain-service/ballerina/resources/admin-api.yaml [hunk 1]
```diff
@@ -2719,6 +2719,193 @@ paths:
           -F "type=WSO2-IS" "https://127.0.0.1:9443/api/am/admin/v3/key-managers/discover"'
       operationId: getWellKnownInfoKeyManager
 
+######################################################
+  # The "Organization" resource API
+######################################################
+  /organization:
```
Comment (CrowleyRajapakse) [inline admin/admin-domain-service/ballerina/resources/admin-api.yaml:2725]: this might need to be organizations, plural

Comment (Krishanx92) [reply admin/admin-domain-service/ballerina/resources/admin-api.yaml]: Fix https://github.com/wso2/apk/pull/630/commits/eaca1b60862435210c947e860d55bb6d711a23aa

File: admin/admin-domain-service/ballerina/resources/admin-api.yaml [hunk 2]
```diff
@@ -2722,11 +2722,11 @@ paths:
 ######################################################
   # The "Organization" resource API
 ######################################################
-  /organization:
+  /organizations:
     get:
       tags:
         - Organization (Collection)
-      summary: Get all Organization
+      summary: Get all API Categories
```
Comment (CrowleyRajapakse) [inline admin/admin-domain-service/ballerina/resources/admin-api.yaml:2729]: API Categories -> Organisations, there are few places changed with new commit as well, shall we fix?

**Review by CrowleyRajapakse — APPROVED** [review]

File: admin/admin-domain-service/ballerina/resources/admin-api.yaml [hunk 3]
```diff
@@ -2719,6 +2719,193 @@ paths:
...
+          $ref: '#/components/responses/BadRequest'
+        404:
+          $ref: '#/components/responses/NotFound'
+      security:
+        - OAuth2Security:
+            - apim:admin
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -X GET -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+          "https://127.0.0.1:9443/api/am/admin/organization/0a043c2b-ee75-4ef3-9e1c-fc2610ccfa8b"'
```
Comment (AmaliMatharaarachchi) [inline admin/admin-domain-service/ballerina/resources/admin-api.yaml:2906]: curl example is invalid

Comment (Krishanx92) [reply admin/admin-domain-service/ballerina/resources/admin-api.yaml]: Fix https://github.com/wso2/apk/pull/630/commits/18c4a6d31a2cecda8c33bf21d01ff675e68bbf42

**Review by CrowleyRajapakse — APPROVED** [review]

**Comment by Krishanx92** [comment]
Fix https://github.com/wso2/apk/issues/653

**Pull request merged**